sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -980,7 +980,7 @@ class JoinSuite extends QueryTest with SharedSQLContext {
 
     val left = Seq((1, 2), (2, 3)).toDF("a", "b")
     val right = Seq((1, 2), (3, 4)).toDF("c", "d")
-    val df = left.join(right, pythonTestUDF($"a") === pythonTestUDF($"c"))
```

Comment:
Just want to point out that `$"col"` is recommended over `df("col")`

Reply:
Yea .. I would like to eventually support that case in the near future ...